packages/react-aria-components/src/useDragAndDrop.tsx

```diff
@@ -59,7 +59,7 @@ interface DropHooks {
   useDroppableCollection?: (props: DroppableCollectionOptions, state: DroppableCollectionState, ref: RefObject<HTMLElement | null>) => DroppableCollectionResult,
   useDroppableItem?: (options: DroppableItemOptions, state: DroppableCollectionState, ref: RefObject<HTMLElement | null>) => DroppableItemResult,
   useDropIndicator?: (props: AriaDropIndicatorProps, state: DroppableCollectionState, ref: RefObject<HTMLElement | null>) => DropIndicatorAria,
-  renderDropIndicator?: (target: DropTarget) => JSX.Element,
+  renderDropIndicator?: (target: DropTarget, keys: Set<Key>, draggedKey?: Key) => JSX.Element,
```

Comment:
I think the API is fine, its consistent with things like `renderPreview`. One small nit would be to perhaps make `keys` optional instead since we won't have access to keys/draggedKeys if the user is dropping from Finder for example

Reply:
I was debating the same thing, and I think I agree. Just updated.